utils/manifest-utils/src/util.rs

```diff
@@ -323,6 +319,14 @@ pub enum KeystoreRemoveResult {
     Removed { removed: Vec<X509> },
 }
 
+/// Calculates Md5 of `txt` and returns first 8 characters.
+pub fn str_to_short_hash<STR: AsRef<[u8]>>(txt: STR) -> String {
```

Comment:
nitpick: considering that you don't need this type to be named, you could just do `pub fn str_to_short_hash(txt: impl AsRef<[u8]>) -> String`.

Reply:
I will change it, but I used to avoid anonymous generic parameters in public functions because someone may want to use it for different types and they will need to change it back into a named generic.
Done.